index.js

```diff
@@ -55,6 +55,9 @@ function ratelimit(opts) {
 
     var delta = (limit.reset * 1000) - Date.now() | 0;
     var after = limit.reset - (Date.now() / 1000) | 0;
+    if (opts.throwable) {
+      throw opts.throwable(after,ms(delta, { long: true }));
```

Comment:
this allows for customized errors.in case you do not want to re-throw another error to meet application's standards